float64[3] linPos
float64[3] linVel
float64[3] linAcc
float64 msgTime